strings.xml

```diff
@@ -460,26 +460,26 @@
     <string name="show_stdout_summary">Dialog with an auto-updating flashing log. Particularly useful for debugging</string>
     <string name="show_stdout_title">Show logs when flashing</string>
     <string name="show_amp_voltage_title">Show charging amperage and wattage</string>
-    <string name="show_amp_voltage_tip">Amperage and voltage depends on the charger &amp; cable that you\'re using. These charging values can also be affected by your usage, device heat and/or battery temperature. This feature simply reads the values from the Kernel, but they might not a 100% accurate representation of the real electricity current so take these as an estimation.\nIf for some reason these stats don\'t show up then it\'s likely your device has different paths. Contact me and we\'ll figure it out together</string>
+    <string name="show_amp_voltage_tip">Amperage and voltage depends on the charger &amp; cable that you\'re using. These charging values can also be affected by your usage, device heat and/or battery temperature. This feature simply reads the values from the kernel, but they might not a 100% accurate representation of the real electricity current, so take these as an estimation.\nIf for some reason these stats don\'t show up, then it\'s likely your device has different paths. Contact me and we\'ll figure it out together</string>
     <string name="core_control_title">Cores control</string>
     <string name="online">Online</string>
     <string name="offline">Offline</string>
-    <string name="cant_offline_cpu0">You can\'t offline CPU0 because it\'s essential and vital to the device\'s functions</string>
-    <string name="cores_control_tip">Turning off cores is a quick and dirty way to save some power. Be wary the device will likely suffer some performance loss exponentially to the amount of cores going offline.\nAlso some chips will have restrictions, for example on big.Little devices, the big cluster will always need one core online</string>
+    <string name="cant_offline_cpu0">You can\'t put offline CPU0 because it\'s essential and vital for device\'s functions</string>
```

Comment:
```suggestion
    <string name="cant_offline_cpu0">You can\'t put CPU0 offline because it\'s essential and vital for device\'s functions</string>
```